### ISSUE TITLE
LRO metadata and response types that aren't imported cause generator crashes

### ISSUE THREAD

**Issue by software-dov** [googleapis/gapic-generator-python#318]
Example:
```proto
// bivalve.proto
// MolluscMetadata lives in mollusc.proto which is not imported
rpc IdentifyBivalve(IdentifyBivalveRequest) returns (google.longrunning.Operation) {
  option (google.longrunning.operation_info) = {
    response_type: "Mollusc",
    metadata_type: "google.molluscs.v1.MolluscMetadata",
  };
} 
```

Causes the following backtrace:
```
 File "/usr/local/google/home/dovs/code/gapic-generator-python/gapic/schema/api.py", line 546, in _get_methods
    metadata_type=self.api_messages[service_address.resolve(
  File "/usr/local/google/home/dovs/.pyenv/versions/3.8.1/lib/python3.8/collections/__init__.py", line 891, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/usr/local/google/home/dovs/.pyenv/versions/3.8.1/lib/python3.8/collections/__init__.py", line 883, in __missing__
    raise KeyError(key)
KeyError: 'google.molluscs.v1.MolluscsMetadata'
--python_gapic_out: protoc-gen-python_gapic: Plugin failed with status code 1.
```

There are a few possible solutions:
1. Require the corresponding import statement in the proto file.
2. Scour the entire API for LRO response/metadata types instead of just the current view.

The downside of 1 is that it will lead to more "imported but not used warnings". It is tricky to implement 2 because of the way that the generated surface references types when wrapping LROs.

**Comment by software-dov** [comment]
A common case is fixed by #330

**Comment by software-dov** [comment]
This is technically prohibited by [AIP-151](https://aip.dev/151). Closing as kinda fixed, kinda not.
